### ISSUE TITLE
Issue with "box-sizing": Confusing order of examples in css demo.

### ISSUE THREAD

**Issue by urajput63** [mdn/interactive-examples#1874]
MDN URL: https://developer.mozilla.org/en-US/docs/Web/CSS/box-sizing

#### What information was incorrect, unhelpful, or incomplete?
In the CSS Demo: box-sizing, The content-box example is placed in the second row, but It confused me while trying to understand the concept.

#### What did you expect to see?
I thought It would make more sense if the third row was swapped with second row. That way when someone reads the document, it would grow to the person how the property's attributes affect the content, as they go row-by-row looking at the example.

#### Did you test this? If so, how?

<details>
<summary>MDN Content page report details</summary>

* Folder: `en-us/web/css/box-sizing`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/CSS/box-sizing
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/css/box-sizing/index.html
* Last commit: https://github.com/mdn/content/commit/ee55a5f6172d7645991ce29afdeb100a8a872206
* Document last modified: 2021-06-17T02:39:52.000Z

</details>

**Comment by nootz1999** [comment]
@urajput63 I think it is a perfectly fine explanation as:

- The first example showing a box (child container) with a few properties namely content-box and width in a parent container.

- The second example shows the same box (child container) but this time we have applied border and padding to it because of which there is an increment in its total height and total width, resulting it to overflow from its parent container. 

- And the third example shows us the magic of the property **box-sizing: border-box;** 
which means when we set the property box-sizing: border-box; to our child container, it will let you add border and padding to your child container without any increment in total width and height of which values will be remain what you have specified (in this case we have already specified width of the container to be 100%). And the third example shows us the added border and padding to it but this time without overflow from its parent container.

@urajput63 I hope this clarifies your confusion.

@rachelandrew The example is fine but I think the content of the page should be changed to something easy and beginner-friendly.
The content is correct but the way it is explained is not easy for a beginner to understand.
